src/battle_script_commands.c

```diff
@@ -15175,16 +15175,16 @@ static void Cmd_handleballthrow(void)
         u32 odds, i;
         u8 catchRate;
 
-        gLastThrownBall = gLastUsedItem;
-        gBallToDisplay = gLastThrownBall;
+        gLastUsedItem = ItemIdToBallId(gLastUsedItem);
```

Comment:
I don't think it'd be a good idea to replace `gLastUsedItem` here, as there's the mismatch between Item IDs and ball IDs. On the lower sections, I would replace `gLastUsedItem` with `gLastThrownBall`.

Reply:
I made a local variable instead because it was scary to mutate both of `gLastThrownBall` and `gBallToDisplay`.